nss-cohort-36/keahua-arboretum-technical-knockouts
# Sc menu

git fetch --all
git checkout sc-menu

loops and conditionals added to annex_plants to check which habitats classes allow the plant selected by a user. 

to test follow these instructions:

1. Choose "Annex Plants" from main menu
2. Choose either "Mountain" or Grasslands"( these 2 are working currently)
3. Choose "Add a plant to habitat" from displayed menu
4. Choose plant that goes into either Mountain (mountain apple tree) or Grasslands (Silversword)

the instances of the the habitat should appear on the menu.
